target/linux/ath79/image/common-yuncore.mk

```diff
@@ -0,0 +1,11 @@
+define Build/yuncore-tftp-header-ap147
+	( \
+		echo -n -e "YUNCOREsetenv bootcmd \"bootm 0x9f050000 || bootm 0x9fe80000\" &&" \
+			"saveenv && erase 0x9f050000 +0xfa0000 && cp.b 0x800600c0 0x9f050000 0xfa0000" | \
```

Comment:
Just replace 0xfa0000 by "$filesize"

Reply:
It's not that easy I think as we have a 192 bytes TFTP header at the beginning and these should be substracted from the file size. A workaround could be to make the file size check in OpenWrt image generation after adding the TFTP header.

On newer U-Boots one could do `setexpr filesize $filesize - 192`, but here it isn't possible. That's why I'm not sure how to handle it.

Reply:
Can you then use IMAGE_SIZEto build the line 
i.e. -e "YUNCORE...." IMAGE SIZE "...

Reply:
I would not use IMAGE_SIZE, as it is subject of an ongoing discussion and might be changed/removed eventually.